delfin/exporter/prometheus/prometheus.py

```diff
@@ -11,20 +11,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import datetime
+import os
+import pytz
+import six
 
 from oslo_config import cfg
+from oslo_log import log
+from tzlocal import get_localzone
 
-grp = cfg.OptGroup('PROMETHEUS_EXPORTER')
+LOG = log.getLogger(__name__)
 
+grp = cfg.OptGroup('PROMETHEUS_EXPORTER')
+METRICS_CACHE_DIR = '/var/lib/delfin/metrics'
```

Comment:
Should this be defined seperately in both exporter server and promethus files?

Reply:
yes because both code run on different processes .